strava_to_fittrackee/s2f.py

```diff
@@ -1061,7 +1062,10 @@ def sync_strava_with_fittrackee():
                 act = strava.create_activity_from_strava(a, get_streams=True)
                 if act.lat == [None] and act.long == [None]:
                     # we don't have any GPS data, so do manual activity
-                    fittrackee.upload_no_gpx(act)
+                    if act.type in fittrackee.sport_id_map:
```

Comment:
I would suggest putting this check after line 1062, since we probably don't want to upload even if there's GPS data if we don't understand the sport. I would do something like this before line 1063:

```python
if act.type not in fittrackee.sport_id_map:
    logger.warning(f"Activity type {act.type} not recognized in FitTrackee, skipping!")
    i += 1
    continue
```